app/views/application/_footer.html.erb

```diff
@@ -1,19 +1,47 @@
 <footer>
-  <a class="icon-link" href="https://github.com/RefugeRestrooms/refuge_housing">
-    <img alt="Refuge Housing on github" class="icon icon-github" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <a class="icon-link" href="https://twitter.com/refugerestrooms">
-    <img alt="Refuge on twitter" class="icon icon-twitter" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <a class="icon-link" href="https://www.facebook.com/refugerestrooms">
-    <img alt="Refuge on facebook" class="icon icon-facebook" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <a class="icon-link" href="http://blog.refugerestrooms.org/">
-    <img alt="Refuge's blog on tumblr" class="icon icon-tumblr" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <a class="icon-link" href="<%= contact_path %>">
-    <img alt="Email Us" class="icon icon-email" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <br/>
-  <%= "\&copy; copyleft #{Date.today.year} refuge housing.".html_safe %>
+  <div class="social-icons">
```

Comment:
you can put it in under a :data key, or I think you can just put them in and rails is smart enough.